pkg/deployments/tasks/20221207-aave-rebalanced-linear-pool-v3/readme.md

```diff
@@ -1,10 +1,12 @@
-# 2022-11-15 - Aave Rebalanced Linear Pool
+# 2022-12-07 - Aave Rebalanced Linear Pool v3
 
 Deployment of the `AaveLinearPoolFactory`, for Linear Pools with a wrapped aToken. This new deployment includes:
 
 - A fix for the `AaveLinearPool` being susceptible to spoofed revert data from the external call to fetch the wrapped token rate, potentially resulting in manipulation of the exchange rate used by `AaveLinearPoolRebalancer`.
 - A fix for the `AaveLinearPoolRebalancer` to handle tokens which do not allow setting an approval without first setting the approval to zero.
```

Comment:
Also has versioning, the Protocol ID on create, and configurable pause window (though we are deploying it with standard pause durations)

Reply:
The new action ids should have one for `registerProtocolId`